os/arch/arm/src/imxrt/imxrt_qtmr.c

```diff
@@ -77,6 +77,7 @@ static struct imxrt_qtmr_chipinfo_s imxrt_qtmr1_chipinfo = {
 
 };
 
+#ifdef CONFIG_ARCH_CHIP_FAMILY_IMXRT105x
```

Comment:
Why is this necessary? Doesn't 1020 have 4 channels?

Reply:
1020 has only two qtimers available.

Reply:
Could you let me know where did you find that? When I checked docs and source codes, it has 4 for 1020 as well.
IMXRT1020RM.pdf, page 2482
```
48.3 Features
The TMR module design includes these distinctive features:
• Four 16-bit counters/timers
```

Reply:
Ok. I see. I found there is no QTIMER3 and 4 in imxrt102x_config.h